snapd.go

```diff
@@ -422,7 +457,6 @@ func action(ctx *cli.Context) {
 	if autodiscoverPath != "" {
 		log.Info("auto discover path is enabled")
 		paths := filepath.SplitList(autodiscoverPath)
-		c.SetAutodiscoverPaths(paths)
```

Comment:
Correct, but that's going to change. Removing it will just see it get added back in a later date. This should be left in for now.